### DISCUSSION TITLE
Is there a way to submit a template to be included in the Templates directory?

### DISCUSSION THREAD

**Discussion by bmarshall511** [vercel/community#3022] [Help] [answered]
### Summary

Hey there! I was just wondering, is it possible for us to submit our own templates to be included in the Templates directory? I came across [Vercel's blog post](https://vercel.com/blog/introducing-the-vercel-templates-marketplace) about the Templates marketplace and it mentioned they include "content from the community." So, I'm curious about how we can go about submitting our own templates. Any insights on that? Thanks!

### Steps to Reproduce

1. Visit the [Templates directory](https://vercel.com/new/templates)
2. Where do we submit community-built ones?

**Comment by leerob** [comment, marked as answer]
Hey! We'd like to open up our templates marketplace to have community contributions, but it's not ready yet. Stay tuned!

**Reply by Mikalis** [reply]
any update on that @leerob ?

**Reply by amyegan** [reply]
Our templates marketplace is open to community contributions now. If you're still interested in sharing your template, please use the [templates marketplace submission form](https://vercel.com/templates/submit) to send us the details. 🙂

**Comment by Mikalis** [comment]
very cool! Would you also be open to a paid blogpost?

Amy ***@***.***> schrieb am Sa., 13. Apr. 2024, 01:04:
